packages/flutter_markdown/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.4+2
+
+* Pass a default error builder to image widgets.
```

Comment:
Feedback on the wording here is welcome. I'm not happy with it, but didn't come up with something better yet.

Reply:
"Handle invalid image URIs" ?

Reply:
I think if you just change `pass` to `passes`, it'll be fine.